core/src/mos6502.cpp

```diff
@@ -609,6 +613,25 @@ Pipeline Mos6502::create_eor_instruction(Opcode opcode) {
     return result;
 }
 
+Pipeline Mos6502::create_rol_instruction(Opcode opcode) {
+    const MemoryAccess memory_access = get_memory_access(opcode.family);
+    Pipeline result;
+    result.append(create_addressing_steps(opcode.address_mode, memory_access));
```

Comment:
Rol-instructions are read-modify-write according to: http://atarihq.com/danb/files/64doc.txt
We will have to add support for those if needed in create_addressing_steps and create_absolute_indexed_addressing_steps.

Reply:
Ok, need to do my homework a bit more...